pkg/k8shandler/cluster.go

```diff
@@ -207,16 +209,64 @@ func getNodeUpgradeInProgress(cluster *v1alpha1.Elasticsearch) NodeTypeInterface
 	return nil
 }
 
+func setUUIDs(cluster *v1alpha1.Elasticsearch) {
+
+	for index := 0; index < len(cluster.Spec.Nodes); index++ {
```

Comment:
Couldnt we also do this with the `for each` construct?  `for ... range....`

Reply:
unfortunately no, we can't change the slice as we iterate over it with something like `for range`. found that out through testing